### DISCUSSION TITLE
26.3.2 Deriving Error enums

### DISCUSSION THREAD

**Discussion by njr0** [google/comprehensive-rust#1281] [General]
I think a little but more explanation would be helpful here.

I don't think the course has mentioned positional arguments to format strings before, and though I think people will guess that's what `"{0}"` is, it's not immedately apparent, looking at the code, that there's a argument for it to format.

Similarly, I think a few words about `#[error` and `#[from` might be useful.

**Comment by mgeisler** [comment]
Hey @njr0, yeah, we should expand the speaker notes on the page!

In general, I would like to see speaker notes on every page (#1083).

**Reply by mgeisler** [reply]
I created #1285 for this so it becomes more visible. In general, feel free to file issues for concrete problems like this :slightly_smiling_face:
